CHANGES.rst

```diff
@@ -1,15 +1,16 @@
 Changelog
 =========
 
-1.1.0 - unreleased
+2.0 - unreleased
 ------------------
 
-* Update various imports to zope.formlib and require at least Zope 2.13.
+* Update various imports to zope.formlib.
 
-* Deal with forward compatibility with Zope 4.
+* Support Zope 4 only.
```

Comment:
This should be the in the first line as it is the most important change.

Reply:
Changed.